admin/Makefile

```diff
@@ -1,17 +1,13 @@
 DEFAULT_GOAL: help
 
 .PHONY: test
-test: image  ## Run tox inside a docker container
-	docker run --rm securedrop-admin .venv/bin/tox
-
-.PHONY: image
-image: ## Create the docker image to run tests
-	docker build -f Dockerfile -t securedrop-admin .
+test:  ## Run tox
+	bin/dev-shell tox
```

Comment:
So running tox with a single test will cause an opaque failure (starting `ERROR: InvocationError`) due to the fact that in `tox.ini` we have `--fail-under 70` set when we run `coverage`. What do you think about removing it? I like the idea, but I also like the `congratulations :)` dev experience at the end of a successful tox test run

Reply:
removed, let's keep tox smiling :-)